src/encoded/schemas/functional_characterization_experiment.json

```diff
@@ -236,67 +236,60 @@
-            "minItems": 2,
+            "minItems": 1,
             "items": {
                 "title": "Locus",
                 "description": "The gene whose expression or product is under investigation for the functional characterization experiment.",
                 "type": "object",
                 "additionalProperties": false,
-                "anyOf": [
-                    {
-                        "required": ["gene", "gene_expression_percentile"]
-                    },
-                    {
-                        "required": ["gene", "gene_expression_range_minimum", "gene_expression_range_maximum"]
-                    }
-                ],
                 "properties": {
                     "gene": {
                         "title": "Gene",
                         "description": "The gene under investigation.",
                         "type": "string",
                         "linkTo": "Gene"
                     },
-                    "gene_expression_percentile": {
+                    "expression_percentile": {
                         "title": "Gene expression percentile",
                         "description": "Gene expression read-out expressed as percentile.",
                         "type": "integer",
                         "minimum": 0,
                         "maximum": 100
                     },
-                    "gene_expression_range_minimum": {
+                    "expression_range_minimum": {
                         "title": "Gene expression range minimum",
-                        "description": "Gene expression read-out range minimum precentage value.",
+                        "description": "Gene expression read-out range minimum percentage value.",
                         "type": "integer",
                         "minimum": 0,
                         "maximum": 100
                     },
-                    "gene_expression_range_maximum": {
+                    "expression_range_maximum": {
                         "title": "Gene expression range maximum",
-                        "description": "Gene expression read-out range maximum precentage value.",
+                        "description": "Gene expression read-out range maximum percentage value.",
                         "type": "integer",
                         "minimum": 0,
                         "maximum": 100
                     }
-                }
+                },
+                "oneOf": [
+                    {
+                        "examined_loci": ["gene"],
+                        "maxProperties": 1
+                    },
+                    {
+                        "required": ["gene", "expression_percentile"],
+                        "maxProperties": 2
+                    },
+                    {
+                        "required": ["gene", "expression_range_minimum", "expression_range_maximum"],
+                        "maxProperties": 3
+                    }
```

Comment:
will that work if in the future we add another property (that is not strictly required for each locus) - will that mess up the maxProperties?

Reply:
I think that probably would break it, yes. I think part of the issue with the three cases in oneOf is that "gene" may be specified by itself. Once we add survival properties, I think we would no longer allow "gene" to be submitted alone. This should mean the min/max requirements are no longer needed either (as when Bonita originally wrote this section)

Reply:
I think we need to change the dependency. Let's talk about some other way to specify the logic in this dependency?